LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License
+
+Copyright (c) 2016 Thomas Bredin-Grey, Studio Thick Pty Ltd
```

Comment:
I suggest `Copyright (c) 2016 The MetaType Project Authors`

Then as new authors arrive, you don't have to change it, only update AUTHORS